build.sbt

```diff
@@ -1,76 +1,144 @@
 import scalariform.formatter.preferences._
 import ReleaseTransformations._
+import DefaultArtifactVersions._
 
 organization in ThisBuild := "com.velocidi"
 
 scalaVersion in ThisBuild := "2.12.10"
 
+def module(project: Project, moduleName: String, dependencies: sbt.librarymanagement.ModuleID*) =
+  project.in(file(moduleName))
+  .settings(commonSettings: _*)
+  .settings(name := s"apso-$moduleName",
+    libraryDependencies ++= dependencies)
+
 lazy val core = project.in(file("core"))
```

Comment:
Can't the `module` helper be used here?

Reply:
Yes, done on 815fc9f6e11c0251540c4c31a34cbed6f94ebd2b.